doc/content/integrations/webhooks/cloud-integrations/akenza-core/tts-setup/_index.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Creating a Webhook"
+description: ""
+weight: 2
+---
+
+Once you have prepared the setup in Akenza Core, follow this section to create the Webhook integration.
```

Comment:
```suggestion
Once you completed the setup in Akenza Core, follow this section to create the Webhook integration.
```

Reply:
@laurensslats you are creating more typos!

Once you have prepared or once you have completed